Tests/Unit/LoRaWan.Tests.Unit.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Revert unrelated change.

Reply:
As discussed, I can't do much about this in VS. If we want to standardize on the encoding, we should do this in editorconfig. Manual fiddling with this, is not what we should do. Agreed with @atifaziz that he is looking into it.